### DISCUSSION TITLE
How to use .pt models without pip package

### DISCUSSION THREAD

**Discussion by robot-jr** [ultralytics/ultralytics#1850] [Q&A]
Hello
i trained the instance segmentation for a competition that im attending using yolov8n
the model works fine
but my problem is that the ultralytics pip package is not allowed in the submission.py file that i have to send for the compotition
how can i use the best.pt for this
and i tried exporting my model to onnx but the results were not good at all
and i dont want to loose precision in my model

**Comment by dh031200** [comment]
@robot-jr  You can install it as below, not as a pip package.
```bash
git clone https://github.com/ultralytics/ultralytics
cd ultralytics
pip install -e .
```

You can find more details in the [docs](https://docs.ultralytics.com/quickstart/)
